code/addons/viewport/src/types.ts

```diff
@@ -24,5 +24,63 @@ export interface Config {
   disable: boolean;
 }
 
-export type GlobalState = { value: string | undefined; isRotated: boolean };
+export type GlobalState = {
+  /**
+   * When set, the viewport is applied and cannot be changed using the toolbar. Must match the key
+   * of one of the available viewports.
+   */
+  value: string | undefined;
```

Comment:
Why undefined?

Reply:
It's what the type was already, I assume you can specify `isRotated` only and that would work, using the default viewport if not specified